actions/class.Runner.php

```diff
@@ -341,7 +341,7 @@ public function submitItemState()
         $serviceCallId = $this->getRequestParameter('testServiceCallId');
 
         try {
-            $serviceContext = $this->getServiceContext(false);
+            $serviceContext = $this->getServiceContext(true);
```

Comment:
It is related to this issue:

https://oat-sa.atlassian.net/browse/TAO-2333

The response must not be stored after a pause, but the state needs to be stored.

After a terminate, I agree this is useless to store the state.